draft-pauly-quic-datagram.md

```diff
@@ -54,14 +47,14 @@ a QUIC connection.
 
 # Introduction
 
-The QUIC Transport Protocol {{I-D.ietf-quic-transport}} provides a secure,
+The QUIC Transport Protocol {{!I-D.ietf-quic-transport}} provides a secure,
```

Comment:
I'm not familiar with ! and ? syntax, what do they mean?

Reply:
! is normative, ? is informational. Martin's big on it, and it's easier to manage.

Reply:
Awesome, thanks!